cmd/fetcher/app/server.go

```diff
@@ -137,11 +142,13 @@ func Run(logger *zap.Logger) {
 	})
 
 	logger.Info("fetcher ready to receive requests")
-	err = http.ListenAndServe(":8000", &ochttp.Handler{
-		Handler: mux,
-	})
-	if err != nil {
-		log.Fatal(err)
+	if openTracingEnabled {
+		err = http.ListenAndServe(":8000", &ochttp.Handler{
```

Comment:
Same as above, if we openTracingEnabled is set to false, fetcher wont run HTTP server.